### ISSUE TITLE
Important places..

### ISSUE THREAD

**Issue by srhtylmz19** [tolu360/react-native-google-places#279]
is it possible to be able to figure the close important places (hospitals, metro - police stations etc..) out ?

**Comment by ghost** [comment]
> is it possible to be able to figure the close important places (hospitals, metro - police stations etc..) out ?

**Comment by srhtylmz19** [comment]
yes it possible buddy, here is documentation https://developers.google.com/places/web-service/search
